src/SlashCommands.tsx

```diff
@@ -286,6 +287,41 @@ export const Commands = [
         category: CommandCategories.admin,
         renderingTypes: [TimelineRenderingType.Room],
     }),
+    new Command({
+        command: 'jumptodate',
+        args: '<date>',
+        description: _td('Jump to the given date in the timeline'),
+        isEnabled: () => SettingsStore.getValue("feature_jump_to_date"),
+        runFn: function(roomId, args) {
+            if (args) {
+                return success((async () => {
+                    const unixTimestamp = Date.parse(args);
+                    if (!unixTimestamp) {
+                        throw new Error(`Unable to parse given date ${args}`);
```

Comment:
Not the most user-friendly of user-facing errors

Reply:
Good call! Updated to the following but unsure if use that type of language in other errors:

```
throw new Error(`We were unable to understand the given date (${args}). Try using the format YYYY-MM-DD.`);
```